### ISSUE TITLE
Trezor Bridge or Trezor Suite, GPG Verification

### ISSUE THREAD

**Issue by Pitocoin** [trezor/trezord-go#230]
I'd like the Trezor support to official provide SHA256 for the exe files of both Trezor Bridge and Trezor Suite. I'm not a IT professional, but according to my recent studies about it, it is strongly recommended to check if SHA256 of the executable files matches what is provided by the official site, and as far as I know I have not founded this information at trezor.io.

Additionally at trezor-bridge-2.0.27-win32-install file  I could find Pavol Rusnak signature but at Trezor-Suite-21.4.1-win file it doesn't. 

Could you help me out? I'm stuck on that GPG verification. I want to have 100% sure I can download either Trezor Bridge or Trezor Suite and finally start using my product.

**Comment by prusnak** [comment]
Use Trezor Suite downloaded from https://suite.trezor.io - the signature and the link to the signing key is located under the download button. Don't install trezor-bridge separately as it is bundled within the Suite installer.

**Comment by Pitocoin** [comment]
Prusnak, 
I did as you mentioned but it didn't work.
After downloaded Trezor Suite did recognize my trezor, it asked me to install firmware, I did it but afterwards Trezor Suite kept asking me to connect my Trezor on the USB. I connected but Trezor Suite didn't recognize anymore... The Trezor itself turn on but asks me to access trezor.io. When I do it the website claims that I don't have Trezor Bridge.
Considering that you told me not to download both Trezor Suite and Trezor Bridge what should I do?
